tests/endpoints/cherrypick_test_data/test_valid_input.py

```diff
@@ -0,0 +1,115 @@
+import json
+import re
+from http import HTTPStatus
+from typing import List
+from unittest.mock import patch
+
+import pytest
+import responses
+
+from lighthouse.constants.cherrypick_test_data import (
+    CPTD_STATUS_PENDING,
+    FIELD_CRAWLER_RUN_ID,
+)
+from lighthouse.constants.error_messages import ERROR_CRAWLER_HTTP_ERROR
+
+CRAWLER_URL = "http://127.0.0.1:8100/cherrypick-test-data"
+ENDPOINT_PATH = "/cherrypick-test-data"
```

Comment:
Seems like some duplication, you could move `ENDPOINT_PATH` before and create `CRAWLER_URL` using it.

Reply:
They refer to different endpoints.  The Crawler one is an external endpoint.  The second one is the endpoint in this app under test.

Reply:
OK, so you are keeping them separate for clarity sake?

Reply:
I don't think they should be merged.  The endpoints only happen to have the same path.  It's not a guaranteed future certainty.

Reply:
Cool cool.